crawler/crawlutils.py

```diff
@@ -317,7 +317,8 @@ def snapshot_container(
     output_urls = reformat_output_urls(urls, container.short_id,
                                        snapshot_num, overwrite)
 
-    container_crawl_plugins = plugins_manager.get_container_crawl_plugins()
+    container_crawl_plugins = plugins_manager.get_container_crawl_plugins(
+        features=features)
```

Comment:
hmm, I thought the mention of a plugin name inside config.conf itself signified 'enabled'. But it might make sense to add the enabled field, so that the user doesn't have to keep removing/adding plugins and their associated fields.